craft_parts/__init__.py

```diff
@@ -0,0 +1,19 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2021 Canonical Ltd
```

Comment:
nit: on a previous PR, we decided to use `Copyright 2021 Canonical Ltd.`

Reply:
The license text uses both the work and the symbol, however the [copyright office documentation examples](https://www.copyright.gov/comp3/chap2200/ch2200-notice.pdf) use the form you mentioned so I'll change the notices to use it.